spendsterIOS/WalkthroughPageViewController.swift

```diff
@@ -0,0 +1,93 @@
+//
+//  WalkthroughPageViewController.swift
+//  spendsterIOS
+//
+//  Created by Dmytro Holovko on 3/22/19.
+//  Copyright © 2019 KeyToTech. All rights reserved.
+//
+
+import UIKit
+
+protocol WalkthroughPageViewControllerDelegate: class {
+    func didUpdatePageIndex(currentIndex: Int)
+}
+
+class WalkthroughPageViewController: UIPageViewController, UIPageViewControllerDataSource, UIPageViewControllerDelegate {
+    weak var walkthroughDelegate: WalkthroughPageViewControllerDelegate?
+    let pageSubheadings = ["Welcome to Budget Planner", "Save money with ease", ""]
+    let pageText = ["Take control of your money\nand save them by tracking your expenses", "Easily create budgets, and see our\nsuggestions based on your spending", "Take control of your money\nand save them by tracking your expenses."]
+    let pageImages = ["Illustration", "hand-pork", "illustration-3"]
+    var currentIndex = 0
+    override func viewDidLoad() {
```

Comment:
@Thongvor please add empty line

Reply:
the empty line above each line of code here?

Reply:
@Thongvor functions should be separated by empty lines, not fields

Reply:
@bogdanustyak Got it